clients/web/src/views/body/FolderView.js

```diff
@@ -19,7 +19,12 @@ var FolderView = View.extend({
         this.folderEdit = settings.folderEdit || false;
         this.itemCreate = settings.itemCreate || false;
 
+        this.render();
+    },
+
+    render: function () {
         this.hierarchyWidget = new HierarchyWidget({
+            el: this.$el,
```

Comment:
Heh, this is wonky. I did this long long ago out of pure laziness. I guess it persists to this day.